CONTRIBUTING.md

```diff
@@ -0,0 +1,253 @@
+# Contributing to Eth2Signer
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+Welcome to the Eth2Signer repository!  The following is a set of guidelines for contributing to this 
+repo and its packages. These are mostly guidelines, not rules. Use your best judgment, 
+and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+[Code of Conduct](#code-of-conduct)
+
+[I just have a quick question](#i-just-have-a-quick-question)
+
+[How To Contribute](#how-to-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Contribution](#your-first-contribution)
+  * [Pull Requests](#pull-requests)
+  * [Code Reviews](#code-review)
+
+[Style Guides](#style-guides)
+  * [Java Style Guide](#java-code-style-guide)
+  * [Coding Conventions](#coding-conventions)
+  * [Git Commit Messages & Pull Request Messages](#git-commit-messages--pull-request-messages)
+  
+[Pull Request Labels](#pull-request-labels)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Eth2Signer Code of Conduct](CODE-OF-CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior to [private@pegasys.tech].
+
+## I just have a quick question
+
+> **Note:** Please don't file an issue to ask a question.  You'll get faster results by using the resources below.
+
+* [Eth2Signer documentation]
+* [Gitter]
```

Comment:
Are we still using Gitter?

Reply:
I think we are still using Gitter for now. Though if or when a PegaSys discord is created this should probably move.